xbrowse_server/matchmaker/utilities.py

```diff
@@ -29,19 +31,36 @@ def get_all_clinical_data_for_family(project_id,family_id,indiv_id):
             A JSON object as per MME spec of a patient
     """
     project = get_object_or_404(Project, project_id=project_id)
+    
+    #get contact information if available
+    try:
+        seqr_project = SeqrProject.objects.get(name=project_id)
```

Comment:
project = Project.objects.get(project_id=project_id)
seqr_project = project.seqr_project

check logic
should exit with error if project_id not found

Reply:
Done. And swapped in seqr.Dataset to populate values instead of our older settings.constants